frontend/src/App.js

```diff
@@ -2,12 +2,16 @@ import React from "react";
 import "./App.css";
 import DiagramEditor from "./DiagramEditor/DiagramEditor.jsx";
 
-const API_ID = "j6mw8j4i6h";
+import { Route, Switch, withRouter } from "react-router-dom";
 
-export default class App extends React.Component {
+import CreateDiagram from "./CreateDiagram/CreateDiagram";
+import DiagramList from "./DiagramList/DiagramList";
+
+const API_ID = "1vt7e367o8";
```

Comment:
what is this? is it a secret?

Reply:
it's not a secret, it's parc of the API URL